tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
+    "composite": true,
```

Comment:
TL;DR?

cc @mike-north

Reply:
`composite` projects can be used with `tsc --build` for more efficient rebuilds (without `--build` all files are recompiled with every build). It also allows the project to be used as dev time compilation dependency which should make co-developement of broccoli with other broccoli-related projects when using `yarn link` if broccoli is set up as a project reference in that other project's `tsconfig.json` file.

It's a "best practice" to be a composite project.